ati_netcanoem_ft_driver/src/ati_netcanoem_ft_driver_node_main.ros2.cpp

```diff
@@ -0,0 +1,16 @@
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include "ati_netcanoem_ft_driver_node.ros2.hpp"
+
+int main(int argc, char * argv[])
```

Comment:
As noticed in the other PRs, it would be nice to change this to `char** argv` to be consistent with other code

Reply:
Done in https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/pull/8/commits/b14e5fda2c9376b09c46876e3c2ae7ea76e10675.